forest/oak/base.py

```diff
@@ -499,15 +498,14 @@ def run_hourly(
             number of minutes of walking per hour
         cadence_hourly: np.ndarray
             average cadence per hour
-        frequency: Frequency
-            summary statistics format, Frequency class at constants.py
+        freq_value: float
+            frequency value in hours
```

Comment:
This should probably be an `int` since we don't use fractional hours, except for the new minute frequency. Then you could keep using `Frequency` enum here.